### ISSUE TITLE
Make using docker

### ISSUE THREAD

**Issue by dragetd** [borgbase/vorta-go#1]
I assume you are using Mac as your primary development platform. This is why the Linux target spawns a Docker container…? :-P

It might be a more sane setup to try and create a Makefile that builds the application simply for the host you are on. And then dedicated cross-building targets that either use Docker or some go crosscompiling magic. In theory it could be possible to crosscompile with go… but the setup is not really easy. One could ignore the issue and simply use GitHub Actions with runners for Mac/Windows/Linux to build the appropriate binary.

Would you accept PRs reworking the building? :-)

**Comment by m3nu** [comment]
Cross-compiling on the same platform isn't possible when using Qt. Must use Docker.

But PRs are always welcome of course.

**Comment by dragetd** [comment]
I know that CGO is an issue here, but I read about some developments in that area plus https://github.com/therecipe/qt#user-content-deployment-targets looks like it. But I will simply play around some and if I got something, we can talk about the code. :)

I thought about keeping the issue open until I figured out if docker is necessary, but on the other hand, it is less of an 'issue' than more a 'suggestion', thus closing it until later. :-)
